package/boot/uboot-envtools/files/ipq806x

```diff
@@ -56,13 +56,17 @@ qcom,ipq8064-ap148|\
 qcom,ipq8064-db149)
 	ubootenv_add_uci_config $(ubootenv_mtdinfo)
 	;;
+xiaomi,mi-router-hd)
+	ubootenv_add_uci_config "/dev/mtd9" "0x0" "0x10000" "0x20000"
```

Comment:
You can use the Linux DT compatible to avoid having to add it here

Reply:
All xiaomi routers have two sections that store settings for the bootloader.
It is necessary to allow the user to operate the settings in both sections:
```
	ubootenv_add_uci_config "/dev/mtd9" "0x0" "0x10000" "0x20000"
	ubootenv_add_uci_sys_config "/dev/mtd12" "0x0" "0x10000" "0x20000"
```
Example_1: https://github.com/openwrt/openwrt/blob/9b336455663482739868953b8b1bd46e1dd0943f/package/boot/uboot-envtools/files/qualcommax_ipq807x#L49-L56
Example_2: https://github.com/openwrt/openwrt/blob/9b336455663482739868953b8b1bd46e1dd0943f/package/boot/uboot-envtools/files/mediatek_mt7622#L63-L66